src/algorithms/detect_cycle.h

```diff
@@ -0,0 +1,15 @@
+/*
+* Copyright 2018-2019 Redis Labs Ltd. and Contributors
+*
+* This file is available under the Redis Labs Source Available License Agreement
+*/
+
+#pragma once
+
+#include "../graph/query_graph.h"
+
+/* Detects if graph contains a cycle. 
+ * assuming graph is fully connected. */
+bool AcyclicGraph(
```

Comment:
consider the change to `QG_IsAcyclicGraph` or similar. 
no big deal.

Reply:
Renamed to `IsAcyclicGraph`